site/content/contribute/developer-setup.md

```diff
@@ -19,7 +19,9 @@ If you're migrating from before the monorepo see the [migration notes]({{< ref "
 If you're developing plugins, see the plugin [developer setup]({{< ref "/integrate/plugins/developer-setup" >}}) documentation.
 {{</note>}}
 
-# Prerequisites for WSL Development Environment
+# Prerequisites for Windows
+
+If you're using Windows, we recommend using the Windows Subsystem for Linux (WSL) for Mattermost development. Docker, Go, and Node must all be run from within WSL, so you'll need to install them in WSL even if you already have the Windows versions of them installed.
```

Comment:
Ah, I didn't realize that. I'll get rid of Docker from there.

And I agree that we already cover the last bit in the 3rd point, but I think I'm going to leave it since I think it's good to repeat something so important. We've definitely had a few community members run into that with MM development, and I know I've run into it when doing non-MM development on Windows as well